stable/weblate/values.yaml

```diff
@@ -79,14 +79,17 @@ service:
   port: 80
 
 ingress:
-  enabled: false
+  enabled: true
+  className: ""
```

Comment:
This value was added, but it doesn't seem to be used anywhere?

Reply:
added using this var in ingress

Reply:
reverted to false after testing completes successfully